windows/runner/main.cpp

```diff
@@ -0,0 +1,42 @@
+#include <flutter/dart_project.h>
```

Comment:
What's with all these windows related files?

We're not creating a windows application, it's polluting the codebase